Dockerfile.astro

```diff
@@ -4,7 +4,6 @@ WORKDIR /home/bun/app
 
 COPY package.json bun.lockb ./
 COPY packages/astro/package.json ./packages/astro/package.json
-COPY packages/payload/package.json ./packages/payload/package.json
```

Comment:
I know this is the astro dockerfile, but the `install` command is run on root level so it requires all sub-packages. We can remove this only if bun can install a single workspace.